.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: Serverless LocalStack CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  serverless-localstack-test:
+    name: Serverless LocalStack CI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup NodeJS
+        uses: actions/setup-node@v2
+        with:
+          node-version: '12'
+
+      - name: Start LocalStack
+        run: |
+          docker pull localstack/localstack
+          docker run --rm -d -p 4566:4566 localstack/localstack
+          
```

Comment:
it would be cool if we could change this (in a follow up PR) to use the localstack CLI as we do in our sample :-) https://docs.localstack.cloud/ci/github-actions/